src/IncludedCommands/UsefulFun/fly.lua

```diff
@@ -247,42 +267,101 @@ return {
                         for _,Event in pairs(self.Events) do
                             Event:Disconnect()
                         end
+
+                        --Unbind the ContextActionService bind
+                        ContextActionService:UnbindAction("toggleFlight")
```

Comment:
Like the other, changing the camel case to pascal case.
```suggestion
                        ContextActionService:UnbindAction("ToggleFlight")
```